spec/factories.rb

```diff
@@ -465,4 +465,11 @@
     provider
     provider_user
   end
+
+  factory :validation_error do
+    form_object { 'RefereeInterface::ReferenceFeedbackForm' }
+    details { { feedback: 'must be present' } }
```

Comment:
My understanding is the records should look like this:

```suggestion
    details { { feedback: { messages: ['must be present'], value: '' } } }
```

I could be wrong though, can you double check?

Reply:
You're right - will fix that.